librocksdb_sys/build.sh

```diff
@@ -3,10 +3,14 @@
 set -e
 
 con=1
-if [[ -f /proc/cpuinfo ]]; then
-    con=`grep -c processor /proc/cpuinfo`
+if [ "$MAKE_PARALLELISM" ]; then
```

Comment:
I had to add this and set the env var to 1 otherwise g++ would just run out of memory in travis.